packages/application-components/src/components/modal-pages/internals/modal-page.styles.ts

```diff
@@ -17,13 +18,27 @@ const getContainerSize = (size?: TModalPageSize) => {
       return '900px !important';
     case 30:
       return '1200px !important';
-    case 'scale':
-      return '100%';
     default:
       return '100%';
   }
 };
 
+const getTransformationSize = (size?: TModalPageSize) => {
+  switch (size) {
+    case 10:
+      return '600px';
```

Comment:
Since we are repeating these values, what do you think about having an internal mapper object we can reuse here? 

Something like this:
```js
const stylesBySize = {
  '10': {
    width: '600px',
    transitionTime: 200
  },
  '20': {
    width: '900',
    transitionTime: 200
  },
  '30': {
    width: '1200',
    transitionTime: 200
  },
 'scale': {
    width: '600',
    transitionTime: 300
  },
}
```

Reply:
Great idea, pushed a commit for it.